docs/ramalama-show.1.md

```diff
@@ -0,0 +1,26 @@
+% ramalama-show 1
+
+## NAME
```

Comment:
Should this be Show or Inspect?  Inspect would be closer to Docker and Podman use.

Reply:
I agree, `inspect` seems better than `show`. Changed :heavy_check_mark: